lib/lambda/update/updatePackage.ts

```diff
@@ -102,7 +102,7 @@ const sendUpdateIdMessage = async ({
     ...remainingFields
   } = currentPackage._source;
 
-  if (!updatedId) {
+  if (updatedId === currentPackage._id) {
```

Comment:
should this be `if (!updatedId || updatedId === currentPackage._id) {`?

Reply:
Its impossible for updatedId to not exist.  Its set to default to packageId if one is not provided on line 192.